forkit/tests/test_fork.py

```diff
@@ -24,7 +24,7 @@ def test_shallow_fork(self):
         fork2 = self.author.fork(commit=False)
 
         self.assertEqual(fork2.pk, None)
-        self.assertEqual(fork2._commits.related.keys(), ['posts'])
+        self.assertEqual(list(fork2._commits.related.keys()), ['posts'])
```

Comment:
You could do:
```python
self.assertSequenceEqual(fork2._commits.related.keys(), ['posts'])
```

Reply:
Python 3 doesn't like this.

```pytb
======================================================================
FAIL: test_shallow_fork (forkit.tests.test_fork.ForkModelObjectTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/charlie/code/virtualstock/django-forkit/forkit/tests/test_fork.py", line 27, in test_shallow_fork
    self.assertSequenceEqual(fork2._commits.related.keys(), ['posts'])
AssertionError: Sequences differ: dict_keys(['posts']) != ['posts']

Unable to index element 0 of first sequence

- dict_keys(['posts'])
+ ['posts']


```

Reply:
This works:

```diff
-        self.assertEqual(list(fork2._commits.related.keys()), ['posts'])
+        if not hasattr(self, 'assertCountEqual'):
+            # Python 2 compat
+            self.assertItemsEqual(fork2._commits.related.keys(), ['posts'])
+        else:
+            # Python 3
+            self.assertCountEqual(fork2._commits.related.keys(), ['posts'])
```

It's look-before-you-leap, rather than a try/except, but I think I prefer it this way, in this case, as it will make it easier to remove when Python 2 is dropped.

Reply:
Clearly I'm not very decisive today. I've gone with a `try` after all.